src/components/Api.js

```diff
@@ -0,0 +1,72 @@
+export default class Api {
+  constructor(options) {
+    this._url = options.url;
+    this._headers = options.headers;
+    this._response = options.response;
```

Comment:
1) You should implement the method for handling the response as Api class method
2) Function names should start with a verb

```js
_processResponse = (res) => { 
      if (res.ok) { 
        return res.json(); 
      } 
      return Promise.reject(`Error: ${res.status}`); 
}

getInitialCards() { 
  ...
  .then(this._processResponse)
}
```